WTWSMS/common/nicknames/VIET_immersion_nicknames.txt

```diff
@@ -254,13 +267,17 @@ nick_the_anger_filled = {
 
 nick_the_sage = {
 	allow = {
-		religion = buddhist
+		OR = {
+			religion = buddhist
+			religion = melchisedechianism
```

Comment:
It IS an unique and distinct nickname. ONLY Buddhists, Melchisedechians, or Tzadikites can get it. They are the only ones that see mysticism the same.